### ISSUE TITLE
Videos don't load in ipad2 or iPhone

### ISSUE THREAD

**Issue by mgogoulos** [plumi/plumi.app#1397]
**Reported by and on 23 Dec 2011 13:44 UTC**
The graphic goes round and round but nothing happens...

**Comment by mgogoulos** [comment]
**Modified by and on 11 Jan 2012 00:02 UTC**

**Comment by mgogoulos** [comment]
**Comment by mike on 23 Jan 2012 17:57 UTC**
Can you check on staging with an ipad2 now and let me know if it plays?

**Comment by mgogoulos** [comment]
**Comment by and on 25 Jan 2012 02:51 UTC**
Got a hold of one. It works for the front page video (looks beautiful in fact) but it doesn't work on individual video/view pages.

Also the subs button doesn't load either.

**Comment by mgogoulos** [comment]
**Comment by and on 25 Jan 2012 02:53 UTC**
FYI if i click directly on the mp4 download link it plays fine.

**Comment by mgogoulos** [comment]
**Comment by and on 25 Jan 2012 03:00 UTC**
Sorry, just realised I was meant to test on staging. Neither the front page nor video/view pages work there.

If i click the mp4 link however it does play.

**Comment by mgogoulos** [comment]
**Modified by and on 29 Jan 2012 11:26 UTC**

**Comment by mgogoulos** [comment]
**Comment by anna on 2 Feb 2012 04:30 UTC**
Unweb don't have an iPad to test this. However this bug exists on the iPhone too - I think Mike has one?

I think we decided to try a couple of hours to figure this one out - if it is more trouble than this, we should postpone it.

It does seem like quite a priority though in terms of videos working on mobile platforms.

**Comment by mgogoulos** [comment]
**Comment by mike on 3 Feb 2012 13:15 UTC**
Unfortunately my iPhone is having a lot of issues lately :(

Can you check on staging and let me know if it works on iPad now? Try this one:

http://staging.engagemedia.org/Members/IESR/videos/DVD-IESR-AIR-TAHU-INTERNET.MP4/view

**Comment by mgogoulos** [comment]
**Modified by mike on 3 Feb 2012 13:15 UTC**

**Comment by mgogoulos** [comment]
**Comment by anna on 7 Feb 2012 05:20 UTC**
No.. hitting either of the play buttons does nothing unfortunately, nothing is activated.

**Comment by mgogoulos** [comment]
**Comment by and on 9 Feb 2012 04:16 UTC**
As per before works for me on the front page on em.org but not in the video/view page. Clicking the direct link the video does play. I can't test in staging as it's down.

**Comment by mgogoulos** [comment]
**Comment by cpsaltis on 13 Feb 2012 16:05 UTC**
We're not sure what is the problem with that. We do exactly the same in both places so it's not logical to work in one case and not in the other. Unfortunately we can't test, so I suggest to put this back.

**Comment by mgogoulos** [comment]
**Comment by anna on 14 Feb 2012 03:34 UTC**
Ok, though I guess we will just be waiting until you can borrow an iphone or an ipad until we can look at fixing this? Anybody you know happen to have one..?

Or would this be useful?
https://developer.apple.com/programs/ios/develop.html

It is strange that Andrew could not play the videos on view pages, but on the front page. I wonder what's going on there. Neither work on my iPhone.

**Comment by mgogoulos** [comment]
**Modified by vyybau on 14 Feb 2012 12:09 UTC**

**Comment by mgogoulos** [comment]
**Comment by markos on 1 Mar 2012 18:04 UTC**
We did intensive testing with iphone 4, IOS 5.0.1 9A405 on safari and chrome browsers. Videos appear and play well in all cases (index page, video page). The only problem is that subtitles don't load.

Unfortunately it seems that universal subtitles do not play well on iphone, eg


```
https://www.pivotaltracker.com/projects/224127?story_id=9804751
https://www.pivotaltracker.com/story/show/9821733

```

**Comment by mgogoulos** [comment]
**Comment by and on 4 Mar 2012 11:33 UTC**
I can also check this on an actual ipad, is it in staging or production though?

**Comment by mgogoulos** [comment]
**Comment by markos on 4 Mar 2012 21:31 UTC**
on www.engagemedia.org

**Comment by mgogoulos** [comment]
**Comment by anna on 5 Mar 2012 01:37 UTC**
video does not load on index or video page, for me on safari on an iphone 3gs / ios 4. but could be it is just too slow.. 

i'm thinking of upgrading (finally) so can test more when i get a new one.

**Comment by mgogoulos** [comment]
**Comment by and on 5 Mar 2012 03:32 UTC**
Still only loads on the front page for me on an ipad 1 on safari and not on the individual video view pages. I tried with Mercury browser also and got the same results.

**Comment by mgogoulos** [comment]
**Comment by anna on 13 Mar 2012 01:22 UTC**
from Christos "about iphone/ipad playback, the situation here is rather complex. Our code follows the standard and should work without problems. Unfortunately a set of bugs in mediaelement.js, universal subtitles and the differences in each platform (iOS+device combination) make testing and solving the problems almost impossible. The iOS SDK cannot solve the testing issues, but even if it did we'd still have the mediaelement and universal subs problems..."

moving to next milestone to have another look when we are sitting down together with an ipad and iphone.

**Comment by mgogoulos** [comment]
**Modified by anna on 21 Mar 2012 05:28 UTC**

**Comment by mgogoulos** [comment]
**Comment by anna on 25 Jul 2012 13:33 UTC**
Fixed!! in 4.4.
